bcgov/wps
# Story/hfi haida gwaii zone/2794

To test:

Load HFI Calc and select Coastal Fire Centre.
When the default date range (current prep period) is loaded, you'll see 8 planning areas, the second of which is Haida Gwaii, which has one station labelled Honna. The first planning area is Fraser Zone, which has 4 stations, none of which are Honna. I've selected random fire-start values that are non-default so that the HFI request is stored in our database.

Select the date range April 20-25 - I've copied over a stored HFI request that's in prod over to the DB for this PR. You'll see that there is no Haida Gwaii planning area anymore because it's not in the stored request, but the table is not broken anymore! In this case the Honna station should be appearing in the Fraser Zone but isn't - the bugfix isn't ideal but since Honna station doesn't have any wx data available at this time anyway, and it only affects historic prep periods, I'm calling it good enough.



# Test Links:
[Landing Page](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2799.apps.silver.devops.gov.bc.ca/hfi-calculator)
